packages/plugins/src/foundry/options.ts

```diff
@@ -23,6 +23,29 @@ export const resolvePaths = (outPath: string, buildInfoPath: string) => {
   }
 }
 
+export const checkRequiredTomlOptions = (toml: FoundryToml) => {
+  if (toml.alwaysUseCreate2Factory !== true) {
+    throw new Error(
+      'Missing required option in foundry.toml file:\nalways_use_create_2_factory = true\nPlease update your foundry.toml file and try again.'
+    )
+  }
+
+  if (toml.buildInfo !== true) {
+    throw new Error(
+      'Missing required option in foundry.toml file:\nbuildInfo = true\nPlease update your foundry.toml file and try again.'
```

Comment:
Should be `build_info`, not `buildInfo` (although, interestingly, specifying `buildInfo` in the foundry.toml works too)

Reply:
I'm referring to the error string here btw